pkg/core/native/oracle.go

```diff
@@ -170,7 +187,11 @@ func (o *Oracle) PostPersist(ic *interop.Context) error {
 	for i := range reward {
 		o.GAS.mint(ic, nodes[i].GetScriptHash(), &reward[i], false)
 	}
-	return nil
+
+	if len(removedIDs) != 0 && orc != nil {
```

Comment:
No need to check `orc != nil`.

Reply:
I'd leave it as it is, just to be sure in case of future changes.